test/iron-ajax.html

```diff
@@ -50,6 +50,17 @@
                  debounce-duration="150"></iron-ajax>
     </template>
   </test-fixture>
+  <!-- note(rictic):
+       This makes us dependent on a third-party server, but we need to be able
+       to check what headers the browser actually sends on the wire.
+       If necessary we can spin up our own httpbin server, as the code is open
+       source.
+    -->
```

Comment:
We need only the most minimal of servers for these tests. ~20 lines of node should do it if we wanted to run the server ourselves as part of WCT. I don't think it would be unreasonable for WCT to have a little local httpbin, it could be useful beyond just iron-ajax.

There are even a few projects where folks have started on implementing httpbin in node.